docs/source/deployorderer/ordererdeploy.md

```diff
@@ -205,20 +132,13 @@ INFO 019 Starting orderer:
 INFO 01a Beginning to serve requests
 ```
 
-You have successfully started one node, you now need to repeat these steps to configure and start the other two orderers. When a majority of orderers are started, a Raft leader is elected. You should see something similar to the following output:
-```
-INFO 01b Applied config change to add node 1, current nodes in channel: [1] channel=syschannel node=1
-INFO 01c Applied config change to add node 2, current nodes in channel: [1 2] channel=syschannel node=1
-INFO 01d Applied config change to add node 3, current nodes in channel: [1 2 3] channel=syschannel node=1
-INFO 01e raft.node: 1 elected leader 2 at term 11 channel=syschannel node=1
-INFO 01f Raft leader changed: 0 -> 2 channel=syschannel node=1
-```
+You have successfully started one node, you now need to repeat these steps to configure and start the other two orderers. When a majority of orderers are started, a Raft leader is elected.
```

Comment:
I added this to the above section where I had left the response you get when starting a node. Also I deleted this section starting with "when a majority" since I assume this isn't true until the nodes are actually joined to a channel.